### PR TITLE
Add Slack as a rules_xcodeproj user

### PR DESCRIPTION
**Pull request by erikkerber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ rules.
 ## Projects using rules_xcodeproj
 
 - [Envoy Mobile](https://github.com/envoyproxy/envoy-mobile)
+- Slack
```
Comment (brentleyjones) [inline README.md:55]: Do you want to link to your BazelCon talk where you publicly say this?

Comment (erikkerber) [reply README.md:55]: Is there prior art for the format here?

Comment (brentleyjones) [reply README.md:55]: The links in this list are either open source examples of using rules_xcodeproj, or (starting with yours, but others will be adjusted or added later) blog posts, tweets/toots, or talks announcing use.

Comment (erikkerber) [reply README.md:55]: I'll leave it as-is for now. I want to highlight just Slack here, and not anything personal.

**Comment by brentleyjones** [comment]
Can you rebase? There is a conflict with the recent Robinhood addition.

**Review by brentleyjones — APPROVED** [review]

**Pull request merged**